src/instrument.ts

```diff
@@ -4,6 +4,16 @@ import {nodeProfilingIntegration} from '@sentry/profiling-node';
 Sentry.init({
   dsn: process.env.SENTRY_DSN,
   integrations: [nodeProfilingIntegration()],
-  profilesSampleRate: 1,
   tracesSampleRate: 1,
 });
+
+const client = Sentry.getClient();
+
+if(client) {
+  const profilingIntegration = client.getIntegrationByName("ProfilingIntegration");
+
+  if(profilingIntegration) {
+    // @ts-expect-error this is purposefuly not exposed by the SDK for now
+    profilingIntegration._profiler.start();
```

Comment:
Oh interesting, this doesn't require a `profilesSampleRate`?

Reply:
No, we actually rely on it not being set. It's a bit weird from the API pov, but we dont want to apply the sampling rate per chunk.